index.html

```diff
@@ -2247,6 +2247,23 @@ <h1>DID Methods</h1>
       </thead>
 
       <tbody>
+        <tr>
+          <td>
+              did:3:
```

Comment:
```suggestion
              did:3:
```

Hmm... while the DID Method ABNF only requires one character to identify the DID Method, we have traditionally been wary of people squatting on single character DID Methods. There are two issues with single character DID methods:

1. It's really difficult to communicate what the "3 DID Method" is to people not in the know.
2. It's been discussed as bad form to do so, but we never got to the point to prohibit it, rather depending on a discussion w/ the DID Method author to try and convince them to not do single character DIDs.

We may want to bring this up with the DID WG and just make the normative requirement to be at least two characters.

@oed would you be willing to use at least two characters to identify the DID Method? I note that the DID Method is actually called "3id"... what about `did:3id`?

Reply:
Would be happy to change it in principle. However, we've been using this method specific identifier in production for about two years so that is not really possible to change it without breaking the products we provide to our users unfortunately.

Reply:
> However, we've been using this method specific identifier in production for about two years

Can you please point to when this happened and a public system and/or running code that demonstrates a release two years ago? It'll be important when we discuss this in the WG. You're asking us to grandfather a DID Method you didn't register two years ago, so this'll turn into a special case discussion. For that to happen, we need more information for why we're grandfathering your method.

Reply:
Sure @msporny 
We've built an app where users can create profiles. This has been around for a bit over two years: https://3box.io/login
This app is built on top of the 3Box.js SDK: https://github.com/3box/3box-js

3Box widely used across the Ethereum ecosystem, here's a short selection:
* https://audius.co/ - crypto audio streaming service
* https://snapshot.page/ - a governance forums for many decentralized applictions
* https://omen.eth.link/ - a prediction market that uses 3box.js for commenting
* MetaMask - uses 3box.js to back up user data
* https://v1.alchemy.do/daos/ - DAOstack, a governance tool for decentralized organizations

Reply:
Hmm this has come up in the past (e.g. here https://github.com/w3c-ccg/did-spec/issues/160), and I agree with @msporny we should discuss this in the WG.

One option could be to not prohibit this in DID Core (i.e. allow one-character DID methods on the syntax level), but as a policy don't add them to the registry (or set a really high bar for doing so). Remember, DID methods don't necessarily have to be registered here. Although in this special case I wouldn't be opposed to making an exception.